src/common/OverlaySwap.tsx

```diff
@@ -66,7 +66,7 @@ const OverlaySwap = ({
   const pools = useContext(PoolContext);
 
   const { selectedSigner: signer, network: nw } = useContext(ReefSigners);
-  const network:Network = useNetworkDex(nw);
+  const network:DexProtocolv2 = resolveDexConfig(nw);
```

Comment:
don't use plain methods inside React component  cos they're called multiple times. Best to create hook in this case and use it in all other components as well